aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+import flowservice
+import destinationinstanceservice
+import os
+import time
+from utils import Utils
+
+class ExportDatasetToCloudStorage:
```

Comment:
Since this is customer-facing code I would recommend we start at that point taking the following conventions in every PR to avoid tech debt later on:
- Every class should have a docstring description.
- Every function should have a docstring description, and all parameters and return values should be detailed as well in the docstring.
- Let's put type hints on every argument and return value, this makes the code so much easier to read and also helps catch issues at the optimizer level.